src/tree-view/types.js

```diff
@@ -8,7 +8,7 @@ LICENSE file in the root directory of this source tree.
 import * as React from 'react';
 import type { OverrideT } from '../helpers/overrides.js';
 
-export type ReactRefT<T> = {| current: null | T |};
+export type ReactRefT<T> = { current: null | T } | {| current: null | T |};
```

Comment:
Is relaxing the exact constraint sufficient?

Reply:
I'm trying to understand why `{ current: null | T } | {| current: null | T |}` rather than just `{ current: null | T }`. Can someone explain?

Reply:
createRef returns an exact type. useRef returns an inexact type.